app/uploaders/avatar_uploader.rb

```diff
@@ -0,0 +1,17 @@
+class AvatarUploader < CarrierWave::Uploader::Base
+
+  if Rails.env.production? || Rails.env.development?
+    storage :fog
+  else
+    storage :file
+    enable_processing = false
+  end
+
+  def store_dir
+    "uploads/#{model.class.to_s.underscore}/#{mounted_as}/#{model.id}"
+  end
+
+  def default_url(*args)
```

Comment:
Unused method argument - `args`. If it's necessary, use `_` or `_args` as an argument name to indicate that it won't be used. You can also write as `default_url(*)` if you want the method to accept any arguments but don't care about them.